--- a/numpy/core/overrides.py.bak	2022-02-16 21:41:02.193929378 +0300
+++ b/numpy/core/overrides.py	2022-02-16 21:43:58.464631454 +0300
@@ -154,6 +154,9 @@
     Function suitable for decorating the implementation of a NumPy function.
     """
 
+    if dispatcher.__doc__ is None:
+        dispatcher.__doc__ = ""
+
     if not ARRAY_FUNCTION_ENABLED:
         def decorator(implementation):
             if docs_from_dispatcher:
